File: Interview_Prep__App/native_java/src/main/java/com/dreams/androidquizapp/repositories/models/response/AnswersResponse.java
package com.dreams.androidquizapp.repositories.models.response;

import com.dreams.androidquizapp.repositories.models.Answer;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Tc2r on 5/19/2017.
 * <p>
 * Description:
 */

public class AnswersResponse {
    @SerializedName("answers")
    private ArrayList<Answer> answersList;

    public ArrayList<Answer> getAnswersList() {
        return answersList;
    }

}

File: Android_Interview_Prep_App/app/src/main/java/com/dreams/androidquizapp/network/AnswersResponse.kt
package com.dreams.androidquizapp.network

import com.dreams.androidquizapp.models.Answer
import com.google.gson.annotations.SerializedName

data class AnswersResponse (
    @SerializedName("answers")
    var answers: ArrayList<Answer>
)
//{
//    @SerializedName("answers")
//    var answers = ArrayList<Answer>()
//}